packages/mui-base/src/SliderUnstyled/SliderUnstyled.test.tsx

```diff
@@ -144,12 +144,10 @@ describe('<SliderUnstyled />', () => {
     });
   });
 
-  describe('prop: valueLabelDisplay', () => {
-    it('renders a slider', () => {
-      render(<SliderUnstyled value={30} valueLabelDisplay="auto" />);
+  it('renders a slider', () => {
```

Comment:
Should we add test for the `slots.valueLabel` prop?

Reply:
Well, we can; it's always better to add a test. I added it in commit https://github.com/mui/material-ui/pull/35805/commits/6c137463e8bd0030e97772d09e401bfbe3e4b7e5.